syne_tune/optimizer/baselines.py

```diff
@@ -822,14 +826,11 @@ def __init__(
         # searcher, by converting the metrics. Internally, all metrics are
         # minimized
         searcher = MultiObjectiveMultiSurrogateSearcher(
-            config_space=config_space,
-            metric=metric,
-            mode=mode,
```

Comment:
`mode` is not in `searcher_kwargs`, so needs to be kept here.

Reply:
thanks for the catch!